src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.VariableIdentifier.cs

```diff
@@ -24,28 +25,58 @@ public VariableIdentifier(Symbol symbol, int containingSlot = 0)
 
             public bool Exists
             {
-                get { return (object)Symbol != null; }
+                get { Debug.Assert((object)Symbol != null);  return (object)Symbol != null; }
```

Comment:
Nit: if there are multiple statements, please make multiple lines.

Reply:
Does it make sense to assert an expression and then return that same expression? Doesn't that mean that the compilation is in a bad state if this ever returns false?

Reply:
I'll remove this property if there are no assert failures.

---
In reply to: [259071565](https://github.com/dotnet/roslyn/pull/33562#discussion_r259071565) [](ancestors = 259071565)